### PR TITLE
Xylix Patron Miracles

### PR DESCRIPTION
**Pull request by wshuwshuwshi**
## About The Pull Request

Gives the Vicious Mockery miracle to Xylix acolytes at t2 devotion

Creates, and adds a new miracle for Xylix acolytes, "The Wheel"
    Changes the targets fortune for 5 minutes by a random value between -5 and 5. Requires a 3 second charge up, with a five minute cooldown

## Why It's Good For The Game

Gives Xylix Acolytes actual miracles, instead of only having the default miracles

### CODE REVIEW

**Comment by wshuwshuwshi** [comment]
Closed because I made this pr from 3-4 AM and forgot to test it until reminded to do so. Compiles fine

**Comment by wshuwshuwshi** [comment]
Tested on a private server.

File: code/datums/status_effects/neutral.dm [hunk 1]
```diff
@@ -157,3 +157,89 @@
...
+			wheeleffect = 4
+		if(5)
+			owner.change_stat("fortune", 5)
+			to_chat(owner, span_boldnotice("My heart flutters, I feel as though I won the lottery!"))
+			wheeleffect = 5									
+
+/datum/status_effect/wheel/on_remove()
+	. = ..()
+	if(wheeleffect == -5)
+		owner.change_stat("fortune", 5)
```
Comment (EvenInDeathIStillServe) [inline code/datums/status_effects/neutral.dm:220]: Replace all of these ifs with a single
`owner.change_stat("fortune", -wheeleffect)`

File: code/datums/status_effects/neutral.dm [hunk 2]
```diff
@@ -157,3 +157,89 @@
...
+var/wheeleffect
+/datum/status_effect/wheel
+	id = "lucky(?)"
+	status_type = STATUS_EFFECT_UNIQUE
+	effectedstats = list("fortune" = "?")
+	duration = 3000 //five minutes
+	
+/datum/status_effect/wheel/on_apply()
+	. = ..()
+	switch(pick(-5,-4,-3,-2,-1,0,1,2,3,4,5))
```
Comment (EvenInDeathIStillServe) [inline code/datums/status_effects/neutral.dm:171]:
```
wheeleffect = rand(-5,5)
owner.change_stat("fortune", wheeleffect)
switch(wheeleffect)
	if(-5 to -1)
		to_chat(owner, span_boldnotice("My heart sinks, I feel as though I've lost something!"))
	if(0)
		to_chat(owner, span_boldnotice("My heart beats, I feel as though nothing has changed at all..."))
	if(1 to 5)
		to_chat(owner, span_boldnotice("My heart flutters, I feel as though I won the lottery!"))
```

**Comment by Lutowski** [comment]
fortune concept is epic but please compress the if chains & change the switch on line 171 to switch(rand(-5,5)) you don't have to individually list every number

File: code/datums/status_effects/neutral.dm [hunk 3]
```diff
@@ -157,3 +157,89 @@
 	. = ..()
 	if(.)
 		listening_in = tracker
+
+//Xylix Gambling
+var/wheeleffect
```
Comment (EvenInDeathIStillServe) [inline code/datums/status_effects/neutral.dm:162]: Put wheeleffect under the status effect. You've made it a global var here.

File: code/datums/status_effects/neutral.dm [hunk 4]
```diff
@@ -157,3 +157,89 @@
...
+/datum/status_effect/wheel
+	id = "lucky(?)"
+	status_type = STATUS_EFFECT_UNIQUE
+	effectedstats = list("fortune" = "?")
```
Comment (EvenInDeathIStillServe) [inline code/datums/status_effects/neutral.dm:166]: Get rid of this line.

**Comment by wshuwshuwshi** [comment]
Code updated to reflect feedback. Thank you

**Pull request merged**